src/EnergyPlus/SetPointManager.cc

```diff
@@ -8203,7 +8203,7 @@ namespace SetPointManager {
 						HumRatCntrlType = iCtrlVarType_HumRat;
 					} else if ( SchSetPtMgr( SetPtMgrNum ).CtrlTypeMode == iCtrlVarType_MaxHumRat ) {
 						HumRatCntrlType = iCtrlVarType_MaxHumRat;
-					}
+					} else { continue; }
```

Comment:
So this is kinda awkward.  How about just `return`ing in each of the two existing blocks?

```
if ( SchSetPtMgr( SetPtMgrNum ).CtrlTypeMode == iCtrlVarType_HumRat ) {
	return iCtrlVarType_HumRat;
} else if ( SchSetPtMgr( SetPtMgrNum ).CtrlTypeMode == iCtrlVarType_MaxHumRat ) {
	return iCtrlVarType_MaxHumRat;
}
```

Then you don't need the kinda funny `else { continue; }` code there.  

What do you think?